client/src/pages/Register/Login/index.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import {
+  TextField, Button, Alert, FormHelperText,
+} from '@mui/material';
+import axios from 'axios';
+import './style.css';
+
+const Login = () => {
+  const [value, setValue] = useState({ email: '', password: '' });
+  const [errorMessage, setErrorMessage] = useState('');
+  const history = useHistory();
+  const sendRequest = async (e) => {
+    e.preventDefault();
+    try {
+      if (value.email && value.password) {
+        if (value.password.length >= 7) {
+          await axios.post('api/v1/login', value);
+          history.push('/');
+        } else {
+          setErrorMessage('Password length must be more than 7 characters');
+        }
+      } else {
+        setErrorMessage('Email and Password field must be filled');
+      }
+    } catch (error) {
+      setErrorMessage(error.response.data.error.message);
```

Comment:
```suggestion
      statusCode === 500 
      ?setErrorMessage('Something Error Happened) //or redirect to 500 page
      :setErrorMessage(error.response.data.error.message);
```

You have to test the status code here if it validation error the set it if it's another like database errors or anything else must set message like 'Something error happened' not to show to the user